spec/index.bs

```diff
@@ -8766,6 +8766,10 @@ GPUCommandBuffer includes GPUObjectBase;
     ::
         A [=list=] of [=GPU commands=] to be executed on the [=Queue timeline=] when this command
         buffer is submitted.
+
+    : <dfn>\[[renderState]]</dfn>, of type [=RenderState=]
+    ::
+        The current state used by any render pass commands being executed, initially `null`.
```

Comment:
(I thought I had a comment here but maybe I decided this was better while I was working on it, I can't remember.)

This is a little bit of a weird place for this, though I know GPUCommandBuffers are single use so it's technically fine. It seems more natural to put [[renderState]] on the queue - though probably even worse.
Since we're passing it in as an "argument" to all of the queue-timeline steps anyway, and only accessing this field in one place, maybe we can create and destroy it inside of the submit() algorithm instead? (it can't be easily written as nullable if we do that but I think that's OK)